pyro/contrib/gp/models/model.py

```diff
@@ -136,20 +145,32 @@ def set_data(self, X, y=None):
 
         + Batch training on a sparse variational model:
 
+            >>> import pyro
+            >>> import pyro.contrib.gp as gp
+            >>> import pyro.distributions as dist
+            >>> import torch
+
+            >>> X = torch.tensor([[1., 5, 3], [4, 3, 7]])
+            >>> y = torch.tensor([2., 1])
+            >>> kernel = gp.kernels.RBF(input_dim=3)
+            >>> kernel.set_prior("variance", dist.Uniform(torch.tensor(0.5), torch.tensor(1.5)))
+            >>> kernel.set_prior("lengthscale", dist.Uniform(torch.tensor(1.0), torch.tensor(3.0)))
+            >>> optimizer = pyro.optim.Adam({"lr": 0.01})
+
             >>> Xu = torch.tensor([[1., 0, 2]])  # inducing input
             >>> likelihood = gp.likelihoods.Gaussian()
-            >>> vsgp = gp.models.SparseVariationalGP(X, y, kernel, Xu, likelihood)
+            >>> vsgp = gp.models.VariationalSparseGP(X, y, kernel, Xu, likelihood)
             >>> svi = SVI(vsgp.model, vsgp.guide, optimizer, Trace_ELBO())
             >>> batched_X, batched_y = X.split(split_size=10), y.split(split_size=10)
             >>> for Xi, yi in zip(batched_X, batched_y):
             ...     vsgp.set_data(Xi, yi)
-            ...     svi.step()
+            ...     svi.step()  # doctest: +SKIP
 
         + Making a two-layer Gaussian Process stochastic function:
 
-            >>> gpr1 = gp.models.GPRegression(X, None, kernel1, name="GPR1")
-            >>> Z, _ = gp_layer1.model()
-            >>> gpr2 = gp.models.GPRegression(Z, y, kernel2, name="GPR2")
+            >>> gpr1 = gp.models.GPRegression(X, None, kernel1, name="GPR1")  # doctest: +SKIP
+            >>> Z, _ = gp_layer1.model()                                      # doctest: +SKIP
+            >>> gpr2 = gp.models.GPRegression(Z, y, kernel2, name="GPR2")     # doctest: +SKIP
```

Comment:
would these skip comments show up throughout the docs? is there a way to ignore them?

Reply:
> would these skip comments show up throughout the docs?

They wouldn't show up in the docs, don't worry about it.

> Ah, just wonder why we skip them?

`gplayer1` and a couple of other variables aren't defined, but let me know if there's an easy fix and I'll put it in.

Reply:
@neerajprad My bad, it is a mismatch of names. `gpr1` should be `gp_layer1` and `gpr2` should be `gp_layer2`. Would you help me fix it in your next pull request? Thanks in advance!